### PR TITLE
Update releases md

### PR DESCRIPTION
**Pull request by hansvancalster**
Maintenance PR to reflect new guidelines for admins due to automatisation of a release for a new or updated protocol.

### CODE REVIEW

**Comment by hansvancalster** [comment]
@ElsLommelen feel free to not wait for my return to work to improve this as needed and merge this. Maybe you want to merge this in `renv-setup`. Currently it is set to merge into `main`.

File: RELEASES.md [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
 
-After review of a protocol-branch and before each release the following steps are necessary:
+After review of a protocol-branch and before each release the following steps will be run automatically and may require action:
 
-1.  with the protocol-branch checked-out:
+1.  A GitHub Action will run when a pull request is made to the main branch.
 
-    1.  bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`.
-        Some examples:
+    1.  This action will automatically run these checks:
```
Comment (ElsLommelen) [inline RELEASES.md:7]:
```suggestion
    1.  This action will update the version number if needed and automatically run these checks:
```

File: RELEASES.md [hunk 2]
```diff
@@ -1,29 +1,54 @@
...
-        -   an update of the `2020.02` version of a protocol in the same year (fourth release in that year): `2020.02.dev` to `2020.04`
+        1.  `protocolhelper::check_frontmatter()`
 
-    2.  update the `.zenodo.json` file:
+        2.  `protocolhelper::check_structure()`
 
-        -   add new authorships with the aid of `protocolhelper:::update_zenodo()`
+    2.  Check any error messages resulting from these checks and address them.
+        Repeat this step until all checks succeeded.
+        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
```
Comment (ElsLommelen) [inline RELEASES.md:15]:
```suggestion
        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
        Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
        This should not pose a problem for local clones as this change can be automatically included and will not cause conflicts (i.e. `git pull origin main` with a local protocol branch checked out)
```

File: RELEASES.md [hunk 3]
```diff
@@ -1,29 +1,54 @@
...
+
+        1.  the general and protocol-specific tags (see [release model](README.md#release-model)) will be created and added to that merge commit.
+            This will be done by `protocolhelper:::set_tags()`
+
+            1.  general tag: `protocols-<version number>`
+
+            2.  specific tag: `<protocol-code>-<version number>`
+
+        2.  render the new or updated protocol and commit/push this to the `protocols` repo (which contains the rendered protocols that are needed to build the website).
+            The rendering is done by `protocolhelper:::render_release()`.
```
Comment (ElsLommelen) [inline RELEASES.md:44]:
```suggestion
            The rendering is done by `protocolhelper:::render_release()`.
            The same general and protocol-specific tags are added to this commit on `protocols`.
```

**Review by ElsLommelen — COMMENTED** [review]
Some small additions that seemed useful to me to have the whole workflow documented.  (We only recently agreed to also update this version number automatically, so maybe after @hansvancalster made this PR?)

@florisvdh Do you mind checking these changes for spelling as well?  We only merge it to renv-setup, so @hansvancalster can have his final word on the changes there (after trying the workflow himself).

File: RELEASES.md [hunk 4]
```diff
@@ -1,29 +1,54 @@
...
 
-3.  Check if continuous integration proceeded without problems
+        2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-4.  Make a GitHub release from the general tag
+    1..
+    Check if the second Github Action ran without problems.
+    If needed, admins should address any problems.
+    Note that during these automatic updates, the two commits will be automatically pushed to the remote.
+    This should not pose a problem for local clones as these changes can be automatically included and will not cause conflicts (i.e. `git pull origin main` with a local protocol branch checked out)
```
Comment (ElsLommelen) [inline RELEASES.md:29]:
```suggestion
    To add these commits to your local clone, press pull in the RStudio Git Pane.
```
I suppose also repo-admins don't need this long description to remember they have to pull?  ;-)

File: RELEASES.md [hunk 5]
```diff
@@ -1,29 +1,57 @@
 # Creating a new release (guidelines for repo-admins)
 
-After review of a protocol-branch and before each release the following steps are necessary:
+After review of a protocol-branch and before each release the following steps will be run automatically and may require action:
 
-1.  with the protocol-branch checked-out:
+1.  A GitHub Action will run when a pull request is made to the main branch.
```
Comment (florisvdh) [inline RELEASES.md:5]:
```suggestion
1.  A GitHub Action will run when a pull request to the main branch is made or updated.
```

File: RELEASES.md [hunk 6]
```diff
@@ -1,29 +1,57 @@
...
-        -   an update of the `2020.05` protocol in 2021 (first release in that year): `2020.05.dev` to `2021.01`
-        -   an update of the `2020.02` version of a protocol in the same year (fourth release in that year): `2020.02.dev` to `2020.04`
+        1.  `protocolhelper::check_frontmatter()`
 
-    2.  update the `.zenodo.json` file:
+        2.  `protocolhelper::check_structure()`
 
-        -   add new authorships with the aid of `protocolhelper:::update_zenodo()`
+    2.  Check any error messages resulting from these checks and address them.
+        Repeat this step until all checks succeeded.
```
Comment (florisvdh) [inline RELEASES.md:14]:
```suggestion
        Repeat this step until all checks succeed.
```

File: RELEASES.md [hunk 7]
```diff
@@ -1,29 +1,57 @@
...
+        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
+        Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
+        If this commit is added, press pull in the RStudio Git Pane to add it to your local clone.
```
Comment (florisvdh) [inline RELEASES.md:17]: Not all admins will use the RStudio buttons to control git.

```suggestion
        If this commit is added in the remote, pull it into your local clone.
```

File: RELEASES.md [hunk 8]
```diff
@@ -1,29 +1,57 @@
...
 
-    3.  update the repo `NEWS.md` file
+2.  A second GitHub Action will be run when a pull request to be merged into the main branch *upon approval*.
```
Comment (florisvdh) [inline RELEASES.md:19]:
```suggestion
2.  A second GitHub Action will be run when a pull request to be merged into the main branch *is approved*.
```

At least I think this is what is meant.

File: RELEASES.md [hunk 9]
```diff
@@ -1,29 +1,57 @@
...
 
-3.  Check if continuous integration proceeded without problems
+        2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-4.  Make a GitHub release from the general tag
+    1..
+    Check if the second Github Action ran without problems.
+    If needed, admins should address any problems.
+    Note that during these automatic updates, the two commits will be automatically pushed to the remote.
+    To add these commits to your local clone, press pull in the RStudio Git Pane.
```
Comment (florisvdh) [inline RELEASES.md:31]:
```suggestion
    So pull these commits to get them in your local repo.
```

File: RELEASES.md [hunk 10]
```diff
@@ -1,29 +1,57 @@
...
 
-    1.  general tag: `protocols-YYYY.NN`
-    2.  specific tag: `protocol-code-YYYY.NN`
+        1.  the repo-wide `NEWS.md` file with `protocolhelper:::update_news_release()`
 
-3.  Check if continuous integration proceeded without problems
+        2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-4.  Make a GitHub release from the general tag
+    1..
```
Comment (florisvdh) [inline RELEASES.md:27]:
```suggestion
    1.
```

File: RELEASES.md [hunk 11]
```diff
@@ -1,29 +1,57 @@
...
+
+            1.  Check if this action ran without problems and if needed address them.
+
+            2.  Check success of publication steps at `protocols.inbo.be`
+
+4.  A fourth GitHub Action will run when a tag `protocols-<version number>` is detected.
+    This will trigger a GitHub release in the `protocolsource` repo and in the `protocols` repo.
+    In turn, a GitHub release will trigger the `.zenodo.json` webhook to publish this new version of the repo contents on Zenodo for long-term archiving of the protocols.
+
+    5.  Check success of GitHub releases and publication on Zenodo
```
Comment (florisvdh) [inline RELEASES.md:57]:
```suggestion
    1.  Check success of GitHub releases and publication on Zenodo
```

Github Markdown takes into account the actual number when starting a new numbered list.

**Review by florisvdh — APPROVED** [review]
:+1: Some minor remaining comments below.

**Pull request merged**